src/utils/stream.ts

```diff
@@ -1,20 +1,8 @@
 import { Readable } from 'stream';
 
-import * as merge2 from 'merge2';
+// @ts-expect-error
+import * as ordered from 'ordered-read-streams';
```

Comment:
If this PR ends up being desirable to merge, I can work on getting some sort of `.d.ts` output for ordered-read-streams